src/Domain/FunctionsOfTime/PiecewisePolynomial.cpp

```diff
@@ -112,13 +114,15 @@ void PiecewisePolynomial<MaxDeriv>::update(
           << ", which precedes the previous update time of "
           << deriv_info_at_update_times_.back().time << ".");
   }
-  if (next_expiration_time < expiration_time_) {
+  const double current_expiration_time =
```

Comment:
move up to line ~66 to use it in another `if`?

Reply:
Line 66 is in a different function.